src/pages/instances/CreateInstance.tsx

```diff
@@ -172,7 +182,7 @@ const CreateInstance: FC = () => {
   const notifyCreated = (instanceLink: ReactNode, message?: ReactNode) => {
     toastNotify.success(
       <>
-        Created instance {instanceLink}.{message}
+        Created instance {instanceLink}. {message}
```

Comment:
```suggestion
        Created instance {instanceLink}.{message}
```
There shouldn't be a dangling space here. Message might be null. The only case where it is set, it is wrapped with a p tag, so will jump into another line anyhow.